### DISCUSSION TITLE
New version with php 8.2

### DISCUSSION THREAD

**Discussion by RakeshShrestha** [leokhoa/laragon#460] [Ideas]
when will the new version of laragon will be released

**Comment by jschweighofer** [comment]
I also upgraded PHP to 8.2 and after that, I could not install Laravel via the quick app feature anymore. It was not able to unzip composer packages. The solution was the following:

Open the php.ini file and uncomment ;extension=zip to extension=zip.

Just in case anybody crashes into this.

**Reply by JABirchall** [reply]
You will need to edit more in the default php.ini file for laravel to work from a new php version.

I just copy the php.ini from versions over.

**Reply by haziq812** [reply]
how
